### PR TITLE
feat(docs) :added prisma integration with mercurius

### PR DESCRIPTION
**Pull request by ruheni**
Added Prisma integration to docs to help with #352 🙂

### CODE REVIEW

File: docs/integrations/README.md [hunk 1]
```diff
@@ -4,5 +4,6 @@
 
 - [nexus](/docs/integrations/nexus.md) - Declarative, code-first and strongly typed GraphQL schema construction for TypeScript & JavaScript
 - [TypeGraphQL](/docs/integrations/type-graphql.md) - Modern framework for creating GraphQL API in Node.js, using only classes and decorators
+- [Prisma](/docs/integrations/prisma.md) - Prisma is an open-source ORM for Node.Js and TypeScript.
```
Comment (nikolasburk) [inline docs/integrations/README.md:7]:
```suggestion
- [Prisma](/docs/integrations/prisma.md) - Prisma is an open-source ORM for Node.js and TypeScript.
```

**Review by nikolasburk — CHANGES_REQUESTED** [review]

File: docs/integrations/prisma.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+generator client {
+  provider = "prisma-client-js"
+}
+```
+
+If you wish to use a different database, you can jump to [switching database providers](#switching-database-providers).
+## Create database tables with Prisma Migrate
+
+Add the following models to your `prisma.schema` file:
```
Comment (nikolasburk) [inline docs/integrations/prisma.md:49]:
```suggestion
Add the following model to your `prisma.schema` file:
```

File: docs/integrations/prisma.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+Prisma can be used with plain JavaScript and it embraces TypeScript and provides a level to type-safety that goes beyond the guarantees other ORMs in the TypeScript ecosystem. You can find an in-depth comparison of Prisma against other ORMs [here](https://www.prisma.io/docs/concepts/more/comparisons)
+
+## Installation
+
+Install [Prisma CLI](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-cli) as a development dependency in your project:
+
+```bash
+npm install prisma --save-dev
+npm install @prisma/client
```
Comment (mcollina) [inline docs/integrations/prisma.md:17]: Can you please include the major version? So we know at least which version this was tested on.

**Review by mcollina — COMMENTED** [review]
Thank you @ruheni! Just a nit

**Review by PabloSzx — APPROVED** [review]

File: docs/integrations/prisma.md (hunk 3 above)
Comment (ruheni) [reply docs/integrations/prisma.md:17]: We don't follow semantic versioning, i.e., we introduce breaking changes in minor versions. Prisma is currently at v2.20.1, and we have minor releases every 2 weeks.

Comment (mcollina) [reply docs/integrations/prisma.md:17]: Then include a full version number here, as we would know which version you tested this in :).

Comment (ruheni) [reply docs/integrations/prisma.md]: Here's more details on how we do our [versioning](https://www.prisma.io/docs/about/releases#versioning)

File: docs/integrations/prisma.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+Prisma can be used with plain JavaScript and it embraces TypeScript and provides a level to type-safety that goes beyond the guarantees other ORMs in the TypeScript ecosystem. You can find an in-depth comparison of Prisma against other ORMs [here](https://www.prisma.io/docs/concepts/more/comparisons)
+
+## Installation
+
+Install [Prisma CLI](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-cli) as a development dependency in your project:
+
+```bash
+npm install prisma@^2.20.1 --save-dev
+npm install @prisma/client@^2.20.1
```
Comment (mcollina) [inline docs/integrations/prisma.md:17]: please remove the caret, as this doc is only valid for this minor version

**Pull request merged**